ports/nrf/boards/wrl13990/README.md

```diff
@@ -0,0 +1,47 @@
+# WRL-13990 Board Support
+
```

Comment:
@rolandvs IMO having a README here for a board like this is not the right way to go.  It gives the false impression that this is an official board with support beyond the support that would normally be given for a port (in this case the nrf port).  At this stage I think the best place for a readme, or a "getting started" document for a particular board, would be on the forum, in a personal blog, or kept in a personal repository.

It's great to have support for additional boards, but it should be just the configuration files.

Reply:
I would imagine that every board that is part of the master is supported even if the fileset is brought down to the absolute minimum :-). I would like to suggest the following:

- A page on the micropython wiki would help to "organise" references to websites/blog's etc. for boards that are not part of the official master collection(s).
- The micropython wiki should dedicate one page with a table and links and *maybe* a dedicated page per board 
- A board (mentioned) on the wiki should then at least be (COTS) available for everyone to be listed.
- Any problems/updates need to be resolved by the "owner" and if not maintained regularly it will be removed from the wiki.

In this way there is enough room to add more to the awesome micropython development without overflowing the repo.
 
Roland

Reply:
And of course this is more or less already in place: `https://github.com/micropython/micropython/wiki/Boards-Summary`

So @dpgeorge, @glennrub, @aykevl what would be wise here. I can move all and future  ports (a.k.a. uBlox-NINA-B1 (52832), Mikroe-CLICKER2 (F407), Sparkfun-WRL13990 (52832), ImageCraft-Wifi2Go (F411)) to the wiki and have the board support either in trimmed down version to add to the repo or keep it all on my own GitHub repo? Then I would refrain from issuing PR's and just fill the wiki from time to time...